src/component/materialui/select/index.tsx

```diff
@@ -29,7 +30,10 @@ export function TimelineSelect(props: SelectProps) {
   const timelines = timelineContext?.allTimelines ?? [];
 
   const ventureTimelines = timelines.filter(
-    (timeline) => timeline.ventureId === ventureId
+    (timeline) =>
+      timeline.ventureId === currentVenture.id &&
+      (currentVenture.userRole === "owner" ||
+        (currentVenture.userRole === "member" && timeline.membersWrite))
```

Comment:
Filter out timelines where members aren't allowed to create updates from the timeline selection.